fractal_server/app/runner/__init__.py

```diff
@@ -77,6 +78,10 @@ async def submit_workflow(
         exist, a new dataset with that name is created and within it a new
         resource with the same name.
     """
+    db_sync = next(DB.get_sync_db())
+    job: ApplyWorkflow = db_sync.get(ApplyWorkflow, job_id)  # type: ignore
+    if not job:
+        raise ValueError("Cannot fetch job")
 
     input_paths = input_dataset.paths
     output_path = output_dataset.paths[0]
```

Comment:
Not related to current PR, but I am confused about the presence of `WORKFLOW_DIR` (here) and
```python

    @property
    def job_root_path(self) -> Path:
        settings = Inject(get_settings)
        return settings.RUNNER_ROOT_DIR / f"job_{self.id:06d}"
```
(in `app/models/run.py`).

Reply:
deferred to #334